ci-operator/step-registry/redhat-chaos/cerberus/one-run/kube-burner/redhat-chaos-cerberus-one-run-kube-burner-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: redhat-chaos-cerberus-one-run-kube-burner
+  from_image:
```

Comment:
I'm not sure if this is needed, we can use the existing folders for eachof the workloads